apps/medon-fe/src/pages/RegistrationPage/index.tsx

```diff
@@ -65,8 +68,14 @@ export default function RegistrationPage() {
             <RegistrationConfirmation email={email}></RegistrationConfirmation>
           )}
         </FormContainer>
+        {!isSuccess && (
+          <Footer>
+            <TermsAndConditions />
+            <PrivacyPolicy />
+          </Footer>
+        )}
       </RegContainer>
-      <Sidebar></Sidebar>;
+      <Sidebar></Sidebar>
```

Comment:
why not <Sidebar/>

Reply:
because it's a styled div, and it's not self closing